app/src/main/java/org/p2p/wallet/send/ui/main/SendFragment.kt

```diff
@@ -112,16 +112,28 @@ class SendFragment :
         sendAnalytics.logSendStartedScreen(analyticsInteractor.getPreviousScreenName())
         setupViews()
 
+        // supportFragmentManager for full screen fragments
+        requireActivity().supportFragmentManager.setFragmentResultListener(
```

Comment:
requireActivity().supportFragmentManager O_O ? huh

Reply:
it was working as it before. 
just was removed and all callbacks stopped